modules/helper.rb

```diff
@@ -113,6 +121,21 @@ def self.is_verified?(event, other_user = nil)
       return is_xxx_role?(event, 'vfd', 'Verified', true, user)
     end
 
+    def self.get_roles(event)
+      roles = []
+
+      # I'm sure there's a better way to do this. Please fix it.
+
+      if is_developer?(event) then roles << "dev" end
+      if is_moderator?(event) then roles << "mod" end
+      if is_helper?(event) then roles << "hlp" end
+      if is_donator?(event) then roles << "don" end
+      if is_server_admin?(event) then roles << "adm" end
+      if is_translator?(event) then roles << "trn" end
+      # if is_verified?(event) then roles << "vfd" end
```

Comment:
I noticed you mentioned not doing verified -- I'd recommend you remove this. It compares user, not event and was to check if another user than the current one was verified. The entire function should probably be removed.